doc/schemas/feerates.schema.json

```diff
@@ -15,6 +15,7 @@
       "required": [
         "min_acceptable",
         "max_acceptable",
+        "floor",
```

Comment:
This may have to be rebased and regenerated on top of the #6142 PR since the generated files appear to be mandatory, despite just having been introduced. That could cause a new `cln-rpc` or `cln-grpc` client to fail when coupled with an older version. Good way to test the new inferred code, as it should switch optionality when rebased and regenerated.